src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -174,8 +175,13 @@ private void AssertLicenseLimits(string type, ServerStore serverStore, DatabaseR
                 AssertDocumentsCompressionLicenseLimits(serverStore, databaseRecord, context);
                 break;
             case nameof(PutAiConnectionStringCommand):
+            case nameof(AddEmbeddingsGenerationCommand):
```

Comment:
this looks strange... this not related to this feature? if so where was this before? @grisha-kotler you added that?

Reply:
Before this, if we don't have the Embeddings Generation feature, we would check the number of connection strings that aren't Embedded.
Since we use `PutAiConnectionStringCommand` also for Gen AI, we need to change the implementation.